### PR TITLE
test: 100 coverage

### PR DESCRIPTION
**Pull request by simoneb**
#### Checklist

- [X] run `npm run test` and `npm run benchmark`
- [X] tests and/or benchmarks are included
- [X] documentation is changed or added
- [X] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: plugin.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function onReqHandlerWrapper (fastify) {
 }
 
 function plugin (fastify, options, next) {
-  const secret = options ? options.secret || '' : ''
+  const secret = options.secret || ''
```
Comment (jsumners) [inline plugin.js:51]: For anyone else wondering: Fastify passes in an empty object by default for the options parameter.

**Review by jsumners — APPROVED** [review]
LGTM

File: plugin.js (hunk 1 above)
Comment (simoneb) [reply plugin.js:51]: correct

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**